tests/containers/validate_btrfs.pm

```diff
@@ -30,9 +33,13 @@ sub _btrfs_fi {
 }
 
 sub _sanity_test_btrfs {
-    my ($rt, $dev_path) = @_;
-    my $dockerfile_path = '/root/sle_base_image/docker_build';
-    my $btrfs_head      = '/tmp/subvolumes_saved';
+    my ($rt, $dev_path, $img) = @_;
+    my $dockerfile_path = "~/sle_base_image/docker_build";
+    unless (-d $dockerfile_path) {
```

Comment:
I doubt that it is working as you expecting . In my understanding this code will check if **worker** has directory `$dockerfile_path` and I suspect that you trying to find folder in VM

Reply:
Indeed, you perhaps wanted to do something like

```suggestion
    unless (script_run("test -d $dockerfile_path")) {
```

Reply:
done